web_frontend/osmaxx/utilities/shortcuts.py

```diff
@@ -86,7 +86,8 @@ def inform(self, message_type, message):
 
 
 def get_cached_or_set(
-        cache_string, func, *args, timeout=datetime.timedelta(minutes=15).seconds, on_cache_hit=None, **kwargs):
+        cache_string, func, *args, timeout=datetime.timedelta(minutes=15).total_seconds(), on_cache_hit=None, **kwargs
+):
```

Comment:
I wonder: Should we (in a separate change) allow `get_cached_or_set` to accept a `timedelta` object as `timeout`?